### ISSUE TITLE
Can't upload Draft files when in submission's Copyediting workflow

### ISSUE THREAD

**Issue by defstat** [pkp/pkp-lib#8067]
**Describe the bug**
When the user is in submission's Copyediting workflow stage, and tries to upload Draft Files from the corresponding grid, an error 500 is produced 

```
PHP Fatal error:  Declaration of ManageFinalDraftFilesForm::execute($stageSubmissionFiles, $fileStage = null) must be compatible with ManageSubmissionFilesForm::execute($stageSubmissionFiles = null, $fileStage = null, ...$functionArgs) in ojs-3-dev-3.3.0\lib\pkp\controllers\grid\files\final\form\ManageFinalDraftFilesForm.inc.php on line 39
```

**To Reproduce**
1. Go to Dashboard -> Submissions
2. Click on any submission
3. Click Copyediting workflow stage
4. At 'Draft Files' grid, click 'Upload/Select Files' button
5. See error

**What application are you using?**
OJS 3.3.0 (stable branch)

**PHP version**
PHP 8.0

**Comment by defstat** [comment]
@asmecher @NateWr

**PRs**

**Branch stable_3.3.0**
OJS: pkp/ojs#3449 (Tests Only)
PKP-LIB: #8070

**Comment by asmecher** [comment]
Merged, thanks!

**Comment by asmecher** [comment]
If encountering this issue, you can fix it by applying this change: https://github.com/pkp/pkp-lib/pull/8070/files

**Comment by mhvezina** [comment]
"_If encountering this issue, you can fix it by applying this change in lib/pkp: https://github.com/pkp/pkp-lib/pull/8070/files_"

... and also applying the same kind of fix for the Copyedited section :
controllers/grid/files/copyedit/form/ManageFinalDraftFilesForm.inc.php 
that is: 	

```
function execute($stageSubmissionFiles = null, $fileStage = null, ...$functionArgs) {
		parent::execute($stageSubmissionFiles, SUBMISSION_FILE_COPYEDIT, ...$functionArgs);
	}
```

**Comment by asmecher** [comment]
@mhvezina, do you mean `lib/pkp/controllers/grid/files/copyedit/form/ManageCopyeditFilesForm.inc.php`? I believe that's filed/fixed here: https://github.com/pkp/pkp-lib/issues/8068

**Comment by asmecher** [comment]
(I'll be releasing OJS/OMP/OPS 3.3.0-12 within a few days, which will include fixes for these issues.)

**Comment by mhvezina** [comment]
> ManageCopyeditFilesForm.inc.php

yes "ManageCopyeditFilesForm.inc.php", my bad...

I saw this is fixed in the soon-to-be released 3.3.0.12 but I thought it might be usefull to know for those who will not install this version yet.
